runtime-light/stdlib/string/regex-functions.cpp

```diff
@@ -102,7 +97,29 @@ int64_t skip_utf8_subsequent_bytes(int64_t offset, const std::string_view subjec
   return offset;
 }
 
-bool parse_regex(RegexInfo& regex_info) noexcept {
+bool get_compiled_regex_cache(RegexInfo& regex_info) noexcept {
```

Comment:
This name is a bit confusing. According to its current name, it only tries to take a compiled regex from some cache. I'd suggest renaming it to `compile_regex`